Dockerfile

```diff
@@ -45,22 +118,21 @@ RUN mv node_modules ./migrate/node_modules
 # Copy temp node_modules of app to app folder
 RUN mv _node_modules node_modules
 
-RUN echo '#!/bin/bash\nnode /app/cli/cli.js "$@"' > /usr/bin/homarr
-RUN chmod +x /usr/bin/homarr
+#RUN echo '#!/bin/bash\nnode /app/cli/cli.js "$@"' > /usr/bin/homarr
+#RUN chmod +x /usr/bin/homarr
 RUN cd /app/cli && yarn --immutable
 
-# Expose the default application port
-EXPOSE $PORT
-ENV PORT=${PORT}
+# Root is needed for supervisord
+USER root
 
-ENV DATABASE_URL "file:/data/db.sqlite"
-ENV NEXTAUTH_URL "http://localhost:7575"
-ENV PORT 7575
-ENV NEXTAUTH_SECRET NOT_IN_USE_BECAUSE_JWTS_ARE_UNUSED
+RUN echo '#!/bin/bash\nnode /app/cli/cli.js "$@"' > /usr/bin/homarr
+RUN chmod +x /usr/bin/homarr
 
 HEALTHCHECK --interval=10s --timeout=5s --start-period=5s --retries=3 \
     CMD wget --no-verbose --tries=1 --spider http://localhost:${PORT} || exit 1
 
-VOLUME [ "/app/data/configs" ]
-VOLUME [ "/data" ]
-ENTRYPOINT ["sh", "./scripts/run.sh"]
+#VOLUME [ "/app/data/configs" ]
+#VOLUME [ "/data" ]
```

Comment:
Also remove this if this is no longer needed? I assume you tested with existing instanced and made sure that they won't break with this change?